scripts/mkcloud

```diff
@@ -1030,6 +1030,9 @@ Optional
         /root/.ssh/authorized_keys file and to provisioner barclamp
     want_neutronsles12=1 (default 0)
         if there is a SLE12 node, deploy neutron-network role into the SLE12 node
+    want_sle12_controller=1 (default 0)
+        if there is a SLE12 controller node, deploy SLE12 services ready on SLE12 controller
```

Comment:
Isn't the task of this rather "setup SLE12 controller node" than expecting "if there's is SLE12 controller node" ?